### PR TITLE
try 2

### PR DESCRIPTION
**Pull request by semyondv**
gemfile modified
draw_axes
exeptions

### CODE REVIEW

**Comment by semyondv** [comment]
how to install **libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev** ?
@Wolwer1nE

**Comment by Wolwer1nE** [comment]
Copied from closed PR:

@semyondv 

Travis needs to install dependencies for ruby2d. Copy this version of [.travis.yml](https://raw.githubusercontent.com/mmcs-ruby/silicium/travis-build/.travis.yml) to your branch.

It will fix build, but you will need to fix code issues with Code Climate.

**Comment by semyondv** [comment]
finally

File: lib/plotter.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 require 'silicium'
 require 'chunky_png'
+require 'ruby2d'
+
 
 module Silicium
   module Plotter
-    ##
```
Comment (Wolwer1nE) [inline lib/plotter.rb:6]: ## is a special sequence for rdoc documentation.

**Review by Wolwer1nE — COMMENTED** [review]
There must be tests, docs and an example in readme. 
I will merge this request, but you need to provide tests and docs for your code.

**Pull request merged**